src/test/java/com/google/gcp_variant_transforms/options/VcfToBqContextTest.java

```diff
@@ -43,6 +44,7 @@ public void testVcfContextConstructor_whenCompareFields_thenMatches() throws IOE
     assertThat(vcfToBqContext.getInputFile()).matches(INPUT_FILE);
```

Comment:
I think matches is used for regex pattern matching, I think isEqualTo() would be better?

Reply:
I mean assertThat()...matches() is used for regex pattern matching, I think isEaualTo would be better.